cmd/utils/flags.go

```diff
@@ -1315,7 +1315,10 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 		log.Warn("Disable transaction unindexing for archive node")
 	}
 
-	setEtherbase(ctx, cfg)
+	// only set etherbase if its a zone chain
+	if ctx.GlobalIsSet(RegionFlag.Name) && ctx.GlobalIsSet(ZoneFlag.Name) {
```

Comment:
Seems to be a mismatch between the code and the comment here.

Reply:
Both region and zone flags are only set in zone